Backend/dev/migrations/dynamic-offer-driver-app/0164-special-location-tag.sql

```diff
@@ -0,0 +1,140 @@
+ALTER TABLE atlas_driver_offer_bpp.quote_special_zone ADD column special_location_tag text;
+ALTER TABLE atlas_driver_offer_bpp.estimate ADD column special_location_tag text;
+ALTER TABLE atlas_driver_offer_bpp.driver_quote ADD column special_location_tag text;
+ALTER TABLE atlas_driver_offer_bpp.search_request ADD column special_location_tag text;
+ALTER TABLE atlas_driver_offer_bpp.fare_policy ADD column "description" text;
+CREATE TABLE atlas_driver_offer_bpp.special_location_priority (
```

Comment:
Add proper INDEXES in DB as per the queries being used.

Reply:
Done, added on special_location_priority and fare_product table